### ISSUE TITLE
Sound/music feedback

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/quadrilateral#328]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/879
Not really a bug, but I'll admit I'm having trouble connecting the sounds/music to the shapes/properties in my head. I'm not sure how to look for bugs effectively since I'm having trouble understanding what it is meant to sound like. This may just be me though.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Quadrilateral‬
URL: https://phet-dev.colorado.edu/html/quadrilateral/1.0.0-dev.73/phet/quadrilateral_all_phet.html
Version: 1.0.0-dev.73 2023-01-08 23:48:12 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
I'm also having more trouble understanding the "one unique sound per detected shape" option. For instance, the concave and convex quadrilaterals sound the same to me in this option.

**Comment by jessegreenberg** [comment]
I think this is good feedback for us to review but I am not sure what (if anything) we should do about it now. Sounds are not really changeable at this point. But maybe we could provide a little more context in Preferences Dialog or in teacher materials or something. 

@BLFiedler do you have any thoughts?

**Comment by KatieWoe** [comment]
It may also just be a learning style/individual thing. If it makes sense to others, it may just be how my brain works?

**Comment by terracoda** [comment]
Hey @BLFiedler, did we ever try 1 single sound for each shape?

I am wondering, given the names of the radio buttons if it would make sense from a sound design/learning experience perspective to play a single sound for each shape. Then there would be a built-in way to learn what each sound meant? Just an idea.

Currently, with radio button number 2, "Sounds emphasize independent shapes", it's like each shape has a unique song or tune. Are the combinations random?

Did we ever try the single-sound approach? Would that make for a weird/unhelpful learning experience?

**Comment by BLFiedler** [comment]
I'm open to that change to a single track per shape. I can "see" the difficulty in auditorily distinguishing tracks very quickly. It will likely take some effort to further distinguish shapes with any amount of backing tracks happening. This will, of course, simplify implementation for this sound design.

@jessegreenberg, go ahead and
- [ ] Remove the de-emphasized tracks that play behind each shape's mapped track. Each track should only have the track mapped to it playing. Triangle should be silent like in the layered design. Let's have the track volume set to the same level as the layers maximum.

If it makes sense to simplify the file structure because of this change, go ahead.

Tagging #175 as this change will impact tasks there (and possibly negate them).

**Comment by jessegreenberg** [comment]
Done above, I think. I kind of liked having a mode where all sounds played at once, sad to see it go. Differences between the other sound design may be more difficult to hear now. If you would like to try a happy medium where background sounds still play but more quietly you can adjust this value to be between 0 and 1. Prior to this change it was 0.15.

https://github.com/phetsims/quadrilateral/blob/b9afcc70c193f1dea412e9fb5baf5021a0b72377/js/quadrilateral/view/sound/TracksVolumeEmphasisSoundView.ts#L84-L87

Over to @BLFiedler to review and play with output levels or just close.

**Comment by BLFiedler** [comment]
Yeah, I do agree 😕 . I'll spend a little bit of time with it tomorrow.

**Comment by BLFiedler** [comment]
Alright, I feel the problem actually has more to do with the track selection after some time spent with it. I've made some modifications. I'd like to request a few individual volume tweaks, but I'll do that back in the original issue (#175). Hopefully, the contrast between shapes has improved here. I removed some of the more repetitious high-pitched tracks which I think dominated the background.

Closing.
